bin/set-sqfs-links.sh

```diff
@@ -63,10 +63,10 @@ function call_bmc {
 
 # Finds latest of each artifact regardless of subdirectory.
 echo "Resolving images to boot ... "
-k8s_initrd="$(find ${WEB_ROOT}/ephemeral/data/k8s -name "*initrd*" -printf '%T@ %p\n' | sort -n | tail -1 |  cut -f2- -d" ")"
+k8s_initrd="$(find ${WEB_ROOT}/ephemeral/data/k8s -name "*initrd*" -printf '%T@ %p\n' | grep -v '.txt' | sort -n | tail -1 |  cut -f2- -d" ")"
```

Comment:
There may be other files with word `initrd` in the name, and not ending with `.txt`... Wouldn't it make sense just use more restrictive filter for find, such as `-name "initrd.img-*.xz"`?

Reply:
I think to your point I'd rather remove the leading glob.
The file should always be `^initrd` but the suffix may change.

For example there was a time where CSM was using `initrd`, `initrd.img`, and `initrd.img.xz`.